docs/userguide/networking/get-started-overlay.md

```diff
@@ -0,0 +1,247 @@
+			CPUs: 2
+			Total Memory: 2.043 GiB
+			Name: 30438ece0915
+
+	From this information, you can see that you are running three containers and 2 images on the Master.
+
+3. Create your `overlay` network.
+
+			$ docker network create -d overlay my-net
+
+		You only need to create the network on a single host in the cluster. In this case, you used the Swarm master but you could easily have run it on any host in the cluster.
+
+4. Check that the network is running:
+
+			$ docker network ls
+			NETWORK ID          NAME                DRIVER
+			412c2496d0eb        mhs-demo1/host      host                
+			dd51763e6dd2        mhs-demo0/bridge    bridge              
+			6b07d0be843f        my-net              overlay             
+			b4234109bd9b        mhs-demo0/none      null                
+			1aeead6dd890        mhs-demo0/host      host                
+			d0bb78cbe7bd        mhs-demo1/bridge    bridge              
+			1c0eb8f69ebb        mhs-demo1/none      null     
+
+	Because you are in the Swarm master environment, you see all the networks on all Swarm agents. Notice that each `NETWORK ID` is unique.  The default networks on each engine and the single overlay network.  
+
+5. Switch to each Swarm agent in turn and list the network.
+
+			$ eval $(docker-machine env mhs-demo0)
+
+			$ docker network ls
+			NETWORK ID          NAME                DRIVER
+			6b07d0be843f        my-net              overlay             
+			dd51763e6dd2        bridge              bridge              
+			b4234109bd9b        none                null                
+			1aeead6dd890        host                host                
+
+			$ eval $(docker-machine env mhs-demo1)
+
+			$ docker network ls
+			NETWORK ID          NAME                DRIVER
+			d0bb78cbe7bd        bridge              bridge              
+			1c0eb8f69ebb        none                null                
+			412c2496d0eb        host                host                
+			6b07d0be843f        my-net              overlay        
+
+  Both agents reports it has the `my-net `network with the `6b07d0be843f` id.  You have a multi-host container network running!
+
+##  Run an application on your Network
+
+Once your network is created, you can start a container on any of the hosts and it automatically is part of the network.
+
+1. Point your environment to your `mhs-demo0` instance.
+
+			$ eval $(docker-machine env mhs-demo0)
+
+2. Start an Nginx server on `mhs-demo0`.
+
+		$ docker run -itd --name=web --net=my-net --env="constraint:node==mhs-demo0" nginx
+
+	This command starts a web server on the Swarm master.
+
+3. Point your Machin environment to `mhs-demo1`
+
+		$ eval $(docker-machine env mhs-demo1)
+
+2. Run a Busybox instanceand get the contents of the Ngnix server's home page.
+
+		$ docker run -it --rm --net=my-net --env="constraint:node==mhs-demo1" busybox wget -O- http://web
+		Unable to find image 'busybox:latest' locally
```

Comment:
Perhaps we should omit the "unable to find ... pulling"; yes it's accurate, but may distract from the example